test_util/terraform_init.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+
+# COPS-4642: using fork of terraform-aws-instance as spot instances are unsupported in dcos-terraform.
+dcos_terraform_instance_aws="git::https://github.com/jbarrick-mesosphere/terraform-aws-instance.git?ref=spot-test"
```

Comment:
LGTM. The one question I had was if this repo will be migrated into a mesosphere-specific repo at some point in the future?

Reply:
*Ideally* COPS-4642 will be implemented and we can drop this entirely.

But, if that's not possible for some reason, then we should move this into an official branch or potentially move away from the fork and start the agents in our main.tf and pass their IPs into the universal installer module.

Reply:
Alright. I've filed an IT ticket to have them help us out.

Reply:
Fixed this!